plugins/uberdev/skills/solve-fleet/workflow.js

```
@@ -0,0 +1,616 @@
+  .split(",").map(function (s) { return s.trim(); }).filter(function (s) {
+    return /^[0-9]+$/.test(s);
+  });
+
+const TIERS = { trivial: 1, small: 1, medium: 1, large: 1 };
+// Tiers that get the script-orchestrated design phases. `large` is an alias
+// the triage table may emit; `--full` normalises to medium in the launcher.
+const DESIGN_TIERS = { medium: 1, large: 1 };
+
+function clampInt(v, lo, hi, dflt) {
+  var n = (typeof v === "number") ? v : parseInt(v, 10);
+  if (typeof n !== "number" || n !== n) return dflt; // NaN guard (no isNaN dep)
```

Biome (2.5.5)

[error] This comparison uses the same expression on both sides.

(lint/suspicious/noSelfCompare)